packages/shared/ConsolePatchingDev.js

```diff
@@ -20,71 +20,76 @@ let prevError;
 let prevGroup;
 let prevGroupCollapsed;
 let prevGroupEnd;
+const enableDoubleLogging = global.__REACT_DEVTOOLS_ENABLE_DOUBLE_LOGGING__;
```

Comment:
This is only read once, right? So what if the user switches midway through the session? Seems like it would be nice to for this setting to take effect every time we get to `disabledDepth === 0`. (But we should not change it midway through an existing log.)